open-company/open-company-bot
# Digest design

[Trello](https://trello.com/c/CWoclvdC)

To test:

- Remove all banner images from the staging S3 banner bucket
- Create new posts, some must see, different sections with a Slack user
- Generate a Slack digest
- [x] Are posts in section order?
- [x] Are the design elements from the Trello card done, and matching the Abstract from the trello card?
- [x] Get someone else to comment on your post, does your Slack notification button say "Reply"?
- Review the code
- Merge
- Remove all the banner images from the prod S3 banner bucket
- Deploy
